test/lib/ansible_test/_data/sanity/yamllint/yamllinter.py

```diff
@@ -177,6 +182,8 @@ def get_module_docs(self, path, contents):
 
         docs = {}
 
+        fmt_re = re.compile(r'^# fmt:\s+(\S+)', flags=re.M)
```

Comment:
This will match on any line, not just the first. I don't know if that's intentional or not, but I think we should require the format specifier to be either on the first line or not preceded by anything but whitespace.

Reply:
Yeah, I think that was my intention.  Basically to work with either of the following:

```
EXAMPLES = r'''# fmt: yaml
stuff
'''
```

or

```
EXAMPLES = r'''
# fmt: yaml
stuff
'''
```

I'll make that change, it just didn't occur to me at the time, that I had changed the code, and broke the intent.

Reply:
Addressed.